lib/core/http.js

```diff
@@ -49,6 +49,28 @@ module.exports = (config) => {
     return response.body;
   };
 
+  const pollRelayOutboundConfig = async (
+    debugRequests = false,
+    pollInterval,
+    cb
+  ) => {
+    // setInterval always waits for the delay before firing the first time, but we want to do an initial poll right away
+    setInterval(async () => await cb(), pollInterval);
```

Comment:
How does this handle an error being thrown from the callback? Is it safer to record it like this?
```suggestion
    setInterval(async () => {
      try {
        await cb();
      } catch (err) {
        console.error('An error occurred while attempting to refresh the outbound relay config');
      }
    }, pollInterval);
```

Reply:
Doesn't look like the tests cover an error being thrown currently

Reply:
Refactored and added a test for failing condition